crates/router/src/core/payments/transformers.rs

```diff
@@ -592,6 +594,7 @@ where
                                                     three_ds_method_data: None,
                                                     three_ds_method_url: None,
                                             }),
+                                            poll_config: api_models::payments::PollConfigResponse {poll_id: request_poll_id, delay_in_secs: poll_config.delay_in_secs, frequency: poll_config.frequency},
```

Comment:
Nit pick. 
`PollConfigResponse` can have function named `new` that would accept poll_id and poll_config and return the constructed `PollConfigResponse`.